### PR TITLE
Security Classification FTAs

### PR DESCRIPTION
**Pull request by noronhaa**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-12478

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/aat/resources/features/F-129 - Access Control SC/F-129.feature [hunk 1]
```diff
@@ -0,0 +1,82 @@
+@F-129 @access-control
+Feature: Access Control tests covering Security Classification feature
```
Comment (mattnayler) [inline src/aat/resources/features/F-129 - Access Control SC/F-129.feature:2]: Can you include the feature number in the title.

Can you apply the feature file formatting changes as per RDM-9279: e.g. #1099 (see [/src/aat/resources/features/F-026/F-026.feature](https://github.com/hmcts/ccd-data-store-api/blob/master/src/aat/resources/features/F-026/F-026.feature) as a good example).

**Review by mattnayler — CHANGES_REQUESTED** [review]
I have added some minor comments on formatting and naming convention for users.  Otherwise it all looks very OK. although currently not green in the pipeline yet.

Can you also convert tabs -> spaces in your TD files as the mixed used of both tabs & spaces is resulting in some of these files displaying confusingly in github.

File: src/aat/resources/features/F-129 - Access Control SC/F-129.feature (hunk 1 above)
Comment (mattnayler) [reply src/aat/resources/features/F-129 - Access Control SC/F-129.feature:2]: i.e. add lots of separating lines
*  around feature header```#===========================================```
*  around each scenario```#----------------------------------------------------------------------```
* blank lines to group `Given`, `When` and `Then` elements

File: src/aat/resources/features/F-129 - Access Control SC/SC_Public_Case_Created.td.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{
+	"_guid_": "SC_Public_Case_Created",
+	"_extends_": "Case_Creation_Data_Base",
+
+	"users": {
+    "invokingUser": {
+      "username": "auto.test.cnp@gmail.com",
+      "password": "Pa55word11"
+    }
```
Comment (mattnayler) [inline src/aat/resources/features/F-129 - Access Control SC/SC_Public_Case_Created.td.json:9]: Should this not be : 

```
   "invokingUser": {
      "_extends_" : "CaseworkerAutotest"
   }
```

File: src/aat/resources/features/common/users/BeftaMasterRestrictedCaseworker.td.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "_guid_": "BeftaMasterRestrictedCaseworker",
+
+  "username": "[[$CCD_RESTRICTED_CASEWORKER_EMAIL]]",
+  "password": "[[$CCD_RESTRICTED_CASEWORKER_PASSWORD]]"
```
[the remark this answers is not in the document]
Comment (mattnayler) [reply src/aat/resources/features/common/users/BeftaMasterRestrictedCaseworker.td.json:5]: Perhaps rename to: `RestrictedCaseworker` ?

File: src/aat/resources/features/common/users/BeftaMasterPrivateCaseworker.td.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "_guid_": "BeftaMasterPrivateCaseworker",
+
+  "username": "[[$CCD_PRIVATE_CASEWORKER_EMAIL]]",
+  "password": "[[$CCD_PRIVATE_CASEWORKER_PASSWORD]]"
+}
```
[the remark this answers is not in the document]
Comment (mattnayler) [reply src/aat/resources/features/common/users/BeftaMasterPrivateCaseworker.td.json]: 👍 thanks for updated in 4a18c40cedfb8a4c836b44d86e754064d3157870

File: src/aat/resources/features/common/users/BeftaMasterPublicCaseworker.td.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "_guid_": "BeftaMasterPublicCaseworker",
+
+  "username": "[[$CCD_CASEWORKER_AUTOTEST_EMAIL]]",
+  "password": "[[$CCD_CASEWORKER_AUTOTEST_PASSWORD]]"
```
[the remark this answers is not in the document]
Comment (mattnayler) [reply src/aat/resources/features/common/users/BeftaMasterPublicCaseworker.td.json]: 👍 thanks for updated in 4a18c40cedfb8a4c836b44d86e754064d3157870

File: src/aat/resources/features/F-129 - Access Control SC/F-129.feature (hunk 1 above)
Comment (mattnayler) [reply src/aat/resources/features/F-129 - Access Control SC/F-129.feature]: 👍 thanks for updated in 4a18c40cedfb8a4c836b44d86e754064d3157870

File: src/aat/resources/features/common/users/BeftaMasterRestrictedCaseworker.td.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (noronhaa) [reply src/aat/resources/features/common/users/BeftaMasterRestrictedCaseworker.td.json:5]: ahh yes missed the change for this one when I deleted the other 2 files

File: src/aat/resources/features/F-129 - Access Control SC/SC_Public_Case_Created.td.json (hunk 2 above)
Comment (noronhaa) [reply src/aat/resources/features/F-129 - Access Control SC/SC_Public_Case_Created.td.json:9]: ahh good catch, hardcoded when i was doing some debugging

**Pull request merged**